src/sentry/interfaces/stacktrace.py

```diff
@@ -813,10 +820,19 @@ def get_traceback(self, event, newest_first=None):
         return '\n'.join(result)
 
     def get_culprit_string(self, platform=None):
+        if platform in ('objc', 'cocoa'):
+            strict_culprit = self.extract_culprit_from_frames(reversed(self.frames), platform=platform, strict=True)
```

Comment:
Since we pass `self.frames` in again, can we just restore the original logic of iterating over `reversed(self.frames)` in extract?